lib/logstash/environment.rb

```diff
@@ -77,7 +77,7 @@ def jruby?
     end
 
     def windows?
-      Gem.win_platform?
+      ::Gem.win_platform?
```

Comment:
Don't know, I just fixed this small issue I found while running the PR. I can investigate but I would do it in another PR.